src/widgets/SummaryWidget.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+namespace hipanel\modules\stock\widgets;
+
+
+use Yii;
+use yii\base\Widget;
+
+/**
+ * Class SummaryWidget
+ * @package hipanel\modules\stock\widgets
+ */
+class SummaryWidget extends Widget
+{
+    /**
+     * @var array
+     */
+    public $total_sums;
+
+    /**
+     * @var array
+     */
+    public $local_sums;
+
+    /**
+     * @inheritDoc
+     */
+    public function run()
+    {
+        $locals = $this->getSumsString($this->local_sums);
+        $totals = $this->getSumsString($this->total_sums);
+        return '<div class="summary">' .
+            ($totals ? Yii::t('hipanel:stock', 'TOTAL') . ':' . $totals : null) .
```

Comment:
```suggestion
            ($totals ? Yii::t('hipanel:stock', 'TOTAL: {sum}', ['sum' => $totals]) : null) .
```

Не рви фразы на части, запаковый их в цельное выражение